katharsis-core/src/main/java/io/katharsis/resource/registry/responseRepository/ResponseRepository.java

```diff
@@ -1,5 +1,11 @@
 package io.katharsis.resource.registry.responseRepository;
 
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import io.katharsis.module.ModuleRegistry;
 import io.katharsis.queryParams.QueryParams;
 import io.katharsis.queryspec.DefaultQuerySpecConverter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.queryspec.DefaultQuerySpecConverter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)